src/kOS.Safe/Screen/ScreenBuffer.cs

```diff
@@ -20,12 +22,16 @@ public interface IScreenBuffer
         void AddSubBuffer(SubBuffer subBuffer);
         void RemoveSubBuffer(SubBuffer subBuffer);
         List<char[]> GetBuffer();
+        void AddResizeNotifier(ScreenBuffer.ResizeNotifier notifier);
+        void RemoveResizeNotifier(ScreenBuffer.ResizeNotifier notifier);
+        string DebugDump();
     }
+    
 
     public class ScreenBuffer : IScreenBuffer
     {
-        public const int MAX_ROWS = 36;
-        public const int MAX_COLUMNS = 50;
+        private const int DEFAULT_ROWS = 36;
+        private const int DEFAULT_COLUMNS = 50;
```

Comment:
The above names were changed because, well, it's not the MAX - it's just the starting value and the terminal can be made bigger.